content/docs/error-boundaries.md

```diff
@@ -157,7 +157,7 @@ class MyComponent extends React.Component {
 }
 ```
 
-Note that the above example is demonstrating regular JavaScript behavior and doesn't use error boundaries.
+Note that the above example is demonstrating regular JavaScript behavior and doesn't use error boundaries. However, throwing an error like ```js throw new Error(`ERROR:${this.state.error}`)``` will trigger the surrounding Error Boundary.
```

Comment:
possibly say:

" However, throwing the following error will trigger the surrounding Error Boundary.
```js 
throw new Error(`ERROR:${this.state.error}`)
```
"